cmds/exp/hdparm/hdparm.go

```diff
@@ -90,14 +95,26 @@ func main() {
 	}
 	scuzz.Debug = debug
 
+	timeout, err := time.ParseDuration(*timeoutDuration)
+	if err != nil {
+		log.Fatal(err)
+	}
+
 	for _, n := range flag.Args() {
-		d, err := scuzz.NewSGDisk(n)
+		d, err := scuzz.NewSGDisk(n, func(s *scuzz.SGDisk) {
```

Comment:
can you make that a util function in scuzz? just so it reads nicer here

```go
func WithTimeout(d time.Duration) SGDiskOpt {

}
```

and

```go
scuzz.NewSGDisk(n, scuzz.WithTimeout(timeout))
```

Reply:
done